pkg/ds/farm.go

```diff
@@ -564,6 +565,25 @@ func (dsf *Farm) spawnDaemonSet(
 
 	desiresCh := ds.WatchDesires(quitSpawnCh, updatedCh, deletedCh)
 
+	go func() {
+		aggregateHealth := replication.AggregateHealth(ds.PodID(), *dsf.healthChecker)
```

Comment:
`dsf.healthChecker` is a pointer to an interface. I have a sneaking suspicion it should get changed.